packages/next/src/server/web/spec-extension/image-response.ts

```diff
@@ -0,0 +1,51 @@
+export class ImageResponse {
```

Comment:
Why is this file under server/web/? It is available under the node runtime too

Reply:
it'just put into the same place, aligning with others like `NextResponse`. I think we added them early before they're available for node, we can move them together in the future